tools/sim/bridge/metadrive/metadrive_world.py

```diff
@@ -9,124 +9,215 @@
 from openpilot.tools.sim.bridge.common import QueueMessage, QueueMessageType
 from openpilot.tools.sim.bridge.metadrive.metadrive_process import (metadrive_process, metadrive_simulation_state,
                                                                     metadrive_vehicle_state)
-from openpilot.tools.sim.lib.common import SimulatorState, World
+from openpilot.tools.sim.lib.common import SimulatorState, World , vec3
 from openpilot.tools.sim.lib.camerad import W, H
+from dataclasses import dataclass
+import math
 
 
-class MetaDriveWorld(World):
```

Comment:
You've inflated the diff by changing tabs to 4 spaces, can you reformat the file?

Reply:
apologies , didn't realise tab for 2 spaces . pushed the fix in [1ff2d0c](https://github.com/commaai/openpilot/pull/33836/commits/1ff2d0ce6e23ed10d4fa195c162a74353c8f733d) commit . this makes the diffs more clearer .